documentation/overview/account_setup/registration-and-login.md

```diff
@@ -3,89 +3,57 @@ ospool:
-OSG Connect portal and can carry on with signup.
+## Orientation Meeting
 
-After continuing, and allowing certain permissions in the next screen, you'll be 
-asked to create a profile and save changes. If this works successfully, you should 
-see that your membership to OSG is "pending" on the right hand side of the screen. 
+The orientation meeting generally takes about 20-30 minutes and is a chance to 
+talk about your work, how it will 
+fit on the OSPool, and some practical next steps for getting started. 
 
-### Orientation Meeting
+## Register for an Access Point
 
-Once you've applied to join OSG Connect as described above, an OSG Connect support 
-team member will contact you to arrange an initial orientation meeting. This meeting 
-generally takes about 20-30 minutes and is a chance to talk about your work, how it will 
-fit on the OSG, and some practical next steps for getting started. Some of these 
-next steps are also listed below. 
+Before or during the orientation meeting, you will be prompted to register 
+for an account on a specific OSPool Access Point. The current default is the 
+"OSG Manage" Access Points, with registration via https://registry.cilogon.org. 
+You will be directed to follow instructions on [this page](../ap7-access) to register 
+for an account here. 
 
-### Join a Project
-
-As part of the sign up and meeting process, you'll be asked for information related 
-to your research group so that you can be assigned to an accounting project. For 
-more information about this process, see this guide: [Start or Join a Project in OSG Connect][projects]
-
-### Generate and Add an SSH key
-
-Once your account is created and you're approved, you can generate and upload an 
-SSH key to the OSG Connect website; this key will be duplicated on the OSG Connect 
-login node so that you're able to log in there and submit jobs. 
+## Log In
 
-To see how to generate and add an SSH key, please visit this 
-page: [Step by step instructions to generate and adding an SSH key][ssh-key]
+Once you've gone through the steps above, you should have an account on 
+on OSPool Access Point. 
 
-## Log In
+If your account is on the "OSG Manage" Access Points (registration via 
+https://registry.cilogon.org, accounts on `ap40.uw.osg-htc.org`), follow instructions 
+in this guide for logging in: [Registration and Log In to OSG Manage Access Points](../ap7-access.md)
 
-Once you've gone through the steps above, you should be able to log in to the OSG Connect 
-login node. See the second half of the SSH key guide for details: 
-[How to Log Into the OSG Connect Login Node][ssh-key]
+If your account is on the "OSG Connect" Access Points (registration via 
+https://www.osgconnect.net/, accounts on `login04.osgconnect.net`, `login05.osgconnect.net`), 
+follow instructions 
+in this guide for logging in: [Registration and Log In to OSG Connect Access Points](../connect-access.md)
```

Comment:
I'd delete this text and write something more vague that makes it so we only have to update the connect-access.md and ap7-access.md guides down the road (also avoids duplicate info). 

"Once your account is approved following your meeting with a Facilitator, you will receive an email with your account details and instructions for logging into your account. Shortly after following these instructions, users can login and begin submitting jobs to the Open Science Pool."